### ISSUE TITLE
There seems to be a problem in your code

### ISSUE THREAD

**Issue by yxwang-10** [zyl123456aB/SPSG_attack#1]
Specifically, in line 83 of SPSG/adversary/transfer.py, the tuple (y_t[i].cpu().squeeze(), sg_t[i].cpu().squeeze()) is saved, while in lines 82-84 of SPSG/utils/model.py, it tries to load three elements from the read file, and the labels list does not seem to be used anymore. I wonder what is the reason?

**Comment by zyl123456aB** [comment]
the labels list can be used for stealing the victim model which gives the label as the answer. You can revise code to use label list for this stealing situation. We use the logits instead of labels for stealing the victim model which gives the logits.

**Comment by YYT998-ai** [comment]
请问您修改好这个问题了嘛，我现在也是出现了这个问题
